db/appsock_handler.c

```diff
@@ -380,6 +380,8 @@ static void *fstdump_hndlr(void *arg_)
     return NULL;
 }
 
+int gbl_allow_incoherent_sql = 0;
```

Comment:
Provide an lrl tunable that allows us to interact with a database even though it is incoherent.  It must be set in the lrl file to be useful as you cannot set this or any other tunable for an already incoherent database.  In addition to this setting, a user must issue 'set ignorecoherency on' for this to have an effect.